numpy/_core/_methods.py

```diff
@@ -168,7 +160,8 @@ def _var(a, axis=None, dtype=None, out=None, ddof=0, keepdims=False, *,
         # Compute the mean.
         # Note that if dtype is not of inexact type then arraymean will
         # not be either.
-        arrmean = umr_sum(arr, axis, dtype, keepdims=True, where=where)
+        arrmean = umr_sum(arr, axis, dtype, keepdims=True, where=where,
+                          out=Ellipsis)
```

Comment:
Why not just `out=...`?

Reply:
Yeah, I think some old habit of mine when `...` wasn't actually allowed everywhere.

Anyway, maybe I should just undo the changes here.  Thought I would share them, but the details are pretty tricky for `_methods.py` as mentioned.
So it would be much easier to skip it and just focus on documentation and the few super simple places.

Reply:
Anyway, decided to revert this.  Need to add new tests, but I think this is useful even without the `_methods.py` fixes/changes, so let's untangle them.